node/src/daemon/setup_reporter.rs

```diff
@@ -2437,6 +2517,40 @@ mod tests {
         assert_eq!(result.get("gas-price").unwrap().value, "10".to_string());
     }
 
+    #[test]
+    fn mapping_protocol_is_blanked_out() {
```

Comment:
This test is puzzling. 

I've discovered up there in the production code that your changes in fact don't function and introduce no help for blanking out any of populated parameters in the database. Which was the point of your efforts evidently. 

However then there is this new test here which seems valid and still it's not a duplication of another test. My only way out of this puzzle is now to think that the desired functionality had worked even before you started modifying anything and there had been a void after a test as a proof of it works.

Strange...but I've gone through the upper area twice and couldn't find a clue. I've assumed that the logic, if it exists, must be placed in this file only. Yet, I didn't see anything newly added that would be attached to this kind of logic. Again, it must have been solved even before you came to fix it.

Reply:
This is pretty old, in the scheme of things, and I can't remember exactly why I wrote it.

However, I think I wanted to prove to myself that the standard blanking-out logic works with `mapping-protocol` the way it does with the rest of the parameters.